core/src/main/scala/org/apache/spark/TaskContextImpl.scala

```diff
@@ -126,4 +126,14 @@ private[spark] class TaskContextImpl(
     taskMetrics.registerAccumulator(a)
   }
 
+  private var rddPartitionInfo: TaskOutputId = null
```

Comment:
rename to `taskOutputInfo`, since with the new naming sometimes this is an `RDDPartitionInfo`, sometimes a `SuffleMapOutputId`